src/configuration.ts

```diff
@@ -2,10 +2,12 @@ import { map, pipe, fromPairs, keys } from 'ramda'
 
 interface Configuration {
   readonly frostApiUrl: string
+  readonly mainExplorerUrl: string
```

Comment:
What is this mainExplorerUrl?

Reply:
nvm, its the link to explorer-web so we can give the user a url to to explorer-web until we merge frost-web and explorer-web.